plugins/FeedsProcessor.inc

```diff
@@ -94,10 +94,10 @@ abstract class FeedsProcessor extends FeedsPlugin {
     $languages = array(
       LANGUAGE_NONE => t('Language neutral'),
     );
-    $language_list = language_list('enabled');
-    if (!empty($language_list[1])) {
-      foreach ($language_list[1] as $language) {
-        $languages[$language->language] = $language->name;
+    $language_list = language_list();
+    if (count($language_list)) {
```

Comment:
I have to admit, I'm not 100% sure if that's appropriate - maybe ` > 1`?

Reply:
@indigoxela docs say `language_list()` always returns an array so should be okay. Though maybe it should be `language_list(TRUE)` to be equivalent to d7